Plugin Guide: Notes for --link option (#1416)
apache/cordova-docs: www/docs/en/12.x-2025.01/guide/hybrid/plugins/index.md

```

This creates a symbolic link instead of copying the plugin files, which enables you

> to work on your plugin and then simply rebuild the app to use your changes. You have to add the plugin after you added the platform, otherwise the link will not work. The link will also be lost, if you re-add a platform. If you do this you have to re-add the plugin.

Reviewer: ```suggestion
to work on your plugin and then simply rebuild the app to use your changes. The plugin should be added after the platform is added; otherwise, the link will not work. The link will also be lost if you re-add the platform or restore the project. In that case, you'll need to re-add the plugin to restore the link.
```

How does the above sound?

Reviewer: I used a ";" because they both could be stand-alone sentences.

Restoring a project for example is running the command `cordova prepare` after cloning your project's repo and running `npm i`. Or another example is deleting the existing platfroms and plugins directory and running the `cordova prepare` command.
